docs/src/main/paradox/project/licenses.md

```diff
@@ -1,31 +1,3 @@
 # Licenses
 
-## Akka License
-
-```
-This software is licensed under the Apache 2 license, quoted below.
-
-Copyright 2009-2018 Lightbend Inc. <https://www.lightbend.com>
-
-Licensed under the Apache License, Version 2.0 (the "License"); you may not
-use this file except in compliance with the License. You may obtain a copy of
-the License at
-
-    https://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
-License for the specific language governing permissions and limitations under
-the License.
-```
-
-## Akka Committer License Agreement
-
-All committers have signed this [CLA](https://www.lightbend.com/contribute/cla/akka/current).
-It can be [signed online](https://www.lightbend.com/contribute/cla/akka).
-
-## Licenses for Dependency Libraries
-
-Each dependency and its license can be seen in the project build file (the comment on the side of each dependency):
-@extref[AkkaBuild.scala](github:project/AkkaBuild.scala#L1054) 
+TODO ADD LICENSE HERE
```

Comment:
That's where we'll need a license, I guess?

Reply:
just link to https://www.apache.org/licenses/LICENSE-2.0 and https://www.apache.org/licenses/contributor-agreements.html#clas for now

Reply:
👍 done